runners/wikiConsole/index.ts

```diff
@@ -155,25 +155,33 @@ export class WikiConsole extends WikiRunner {
               return null;
     }
 
-    runNextKatacodaStep(runCommand: RunCommand): RunResult {
+    runDisplayContent(runCommand: RunCommand): RunResult {
         let tempFile = path.join(this.getTempDirectory(), runCommand.command.name + ".md");
         fs.writeFileSync(tempFile, "");
+        let counter = 2;
```

Comment:
You set `counter` to 2 two times. Move it to the beginning of the else-if-block, then you only need it once

Reply:
Removed the second assignment